packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/SyncedRealmTests.kt

```diff
@@ -336,13 +335,12 @@ class SyncedRealmTests {
                 Realm.open(config).use {
                     // Make sure that the test eventually fail. Coroutines can cancel a delay
                     // so this doesn't always block the test for 10 seconds.
-                    delay(10 * 1000)
+                    delay(10_000)
                     channel.send(AssertionError("Realm was successfully opened"))
                 }
             }
 
             val error = channel.receiveOrFail()
-            assertTrue(error is UnrecoverableSyncException, "Was $error")
```

Comment:
Isn't there a test that checks `UnrecoverableSyncException`?

Reply:
That is the only test that mentioned `UnrecoverableSyncException`. I am going to add some tests using `realm_sync_session_handle_error_for_testing`